python/ray/serve/master.py

```diff
@@ -424,7 +426,24 @@ def _scale_replicas(self, backend_tag, num_replicas):
         current_num_replicas = len(self.replicas[backend_tag])
         delta_num_replicas = num_replicas - current_num_replicas
 
+        _, _, replica_config = self.backends[backend_tag]
         if delta_num_replicas > 0:
+            can_schedule = try_schedule_resources_on_nodes(
```

Comment:
Could we add an internal feature flag to turn this off? I just want to make sure if a user wants to use autoscaling we at least have a workaround for them to ignore this even if it isn't a public API.